swarmrl/tasks/searching/_gradient_sensing_vision_cone.py

```diff
@@ -144,8 +150,14 @@ def __call__(self, observable: np.ndarray):
 
         Parameters
         ----------
-        observable : np.ndarray (dim, )
-                Position of the colloid.
+        observable : np.ndarray (dimension, )
+                Observable of a single colloid.
+        colloid : object
```

Comment:
The only reason this task is here is so that when Tobi comes back to look at the code he can make sure nothing is lost from here that is in the new vision cones observable. It should basically never be touched and it will not work with the current RL infrastructure.